Firmware/temperature.cpp

```diff
@@ -2871,30 +2888,33 @@ static bool autotune(int16_t cal_temp)
 
 void temp_model_autotune(int16_t temp, bool selftest)
 {
+    char tm_message[20];
     if(moves_planned() || printer_active()) {
-        SERIAL_ECHOLNPGM("TM: printer needs to be idle for calibration");
+        sprintf_P(tm_message, PSTR("TM: Cal. NOT IDLE"));
+        SERIAL_ECHOLN(tm_message);
+        lcd_setstatuspgm(tm_message);
```

Comment:
`lcd_setstatuspgm` with a RAM resident variable? Shouldn't it have been just `lcd_setstatus`?